package.json

```diff
@@ -53,7 +61,9 @@
     "babel-plugin-import": "1.11.2",
     "canvas-prebuilt": "1.6.11",
     "copy-webpack-plugin": "5.0.3",
+    "documentation": "11.0.0",
     "eslint": "5.16.0",
+    "gh-pages": "2.0.1",
```

Comment:
These are both `devDependencies` or not?

Reply:
Actually they are, the collapsed diff is misleading.